Dockerfile

```diff
@@ -1,8 +1,10 @@
 # VERSION   0.1
-FROM        debian:unstable
+FROM        debian:stable
```

Comment:
Sure, happy to help! :+1:

This is one of those little things that helps smooth out release
transitions.  The best description I ever heard for the "stable" alias was
after I called it a "rolling release" and was told that sure, it's a
rolling release, but it's a very square wheel, so each roll might be a bit
painful. :smile: